### PR TITLE
Tweaks Black Site Prism

### PR DESCRIPTION
**Pull request by Kromkar**
Tweaks Black Site Prism
[tweak]
[vault]

:cl:
 * rscadd: Tweaks Black Site Prism; minor mapping changes, rebalances loot and adds Syndicate Exec mob. Changes Commandant Chuck Sneed to Buck Reed.

### CODE REVIEW

**Comment by Kromkar** [comment]
I figured out the conflict issues and resolved it. This PR includes the removal of the antag vials that caused controversy on the previous unmerged PR.

**Comment by gurfan** [comment]
sneed

**Comment by xeroxemnas** [comment]
why would you remove all the fun stuff.

File: maps/randomvaults/black_site_prism.dmm [hunk 1]
```diff
@@ -1,70 +1,76 @@
...
 "bs" = (/obj/structure/grille,/obj/structure/window/reinforced{dir = 1},/obj/structure/cable/yellow{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/cable/yellow{d1 = 2; d2 = 8; icon_state = "2-8"},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
 "bw" = (/obj/item/weapon/stool/bar,/turf/simulated/floor{dir = 5; icon_state = "cafeteria"},/area/vault/black_site_prism)
-"bz" = (/obj/machinery/light{color = "#FF0000"; dir = 4; light_color = "#FF0000"},/turf/unsimulated/floor{dir = 1; icon_state = "darkred"},/area/vault/black_site_prism)
+"bz" = (/obj/machinery/light{color = "#FF0000"; dir = 4; light_color = "#FF0000"},/obj/structure/sign/pox/red{pixel_x = 32},/turf/unsimulated/floor{dir = 1; icon_state = "darkred"},/area/vault/black_site_prism)
+"bA" = (/mob/living/simple_animal/hostile/humanoid/syndicate/ranged/exec{health = 250; name = "Commandant Chuck Sneed"},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
```
Comment (toomykins) [inline maps/randomvaults/black_site_prism.dmm]: i still hate you

**Comment by Kromkar** [comment]
> why would you remove all the fun stuff.

The adm*ns were very mad that I added a free antag card to the turbohard vault instead of making players do Xenoarch to get it.

**Comment by xeroxemnas** [comment]
Meanwhile you can still get a fucking Overwatch rifle from a vault lmao.

File: maps/randomvaults/black_site_prism.dmm (hunk 1 above)
Comment (Kammerjunk) [inline maps/randomvaults/black_site_prism.dmm]: shit meme

Comment (smalIboss) [inline maps/randomvaults/black_site_prism.dmm:17]: sir i think this pr was meant for the tg repo

**Comment by ShiftyRail** [comment]
We have someone who :

- put their own static in their own vault
- got laughed at for it
- decides to replace said static with a meme out of pettyness
- doesn't understand why he's still being laughed at

Also your emoji'd your own PR

**Comment by kilozombie** [comment]
We don't have any real vault mappers at the moment and this guy is actually putting in the time to make cool maps for us.

You catch more autists with honey. I'm still gonna pester him to test his fucking maps though.

**Comment by BomberBro** [comment]
All he has to do is make the name not a complete meme and I'd absolutely support it, this is a bizarre hill to die on

**Comment by Kromkar** [comment]
> All he has to do is make the name not a complete meme and I'd absolutely support it, this is a bizarre hill to die on

I'm changing the name. I agree it's a bizarre hill to die on, especially since we have a gay japanese leather dungeon gym as a vault.

**Comment by xeroxemnas** [comment]
dont you dare insult gachimuchi you fuck.

**Comment by Eneocho** [comment]
I don't mind the chuck sneed thing, if anything, most people don't mind/care about it.

**Comment by Kromkar** [comment]
> I don't mind the chuck sneed thing, if anything, most people don't mind/care about it.

I thought that too, then I got 7 downvotes because of it

**Comment by toomykins** [comment]
sneed is literally a forced meme that's shouted constantly by like three autists in total.

**Comment by Kromkar** [comment]
There. My sacrilegious joke name has been removed.

**Review by smalIboss — APPROVED** [review]

**Comment by PrimeDSS13** [comment]
hahahaha

**Comment by ShiftyRail** [comment]
Update your changelog and this is mergeable

**Comment by Kromkar** [comment]
> Update your changelog and this is mergeable

Done

**Comment by ShiftyRail** [comment]
You accidentally deleted your vault entirely. Did you actually mean to do that ?

**Comment by ShiftyRail** [comment]
Still at the Commandant Chuck Sneed stage.

**Comment by Kromkar** [comment]
> Still at the Commandant Chuck Sneed stage.

I will take another look tonight. I am growing to hate SmartGit.

**Comment by Dacendeth** [comment]
Might want to remove the camera from the entrance to the vault, or change the network it's on. Very odd for a blacksite to have a camera that people on the station can see.

**Comment by Kurfursten** [comment]
As of latest revision, Commandant Sneed still has command of the site

**Comment by toomykins** [comment]
> Might want to remove the camera from the entrance to the vault, or change the network it's on. Very odd for a blacksite to have a camera that people on the station can see.

the russian vault also has a camera. it's a stylistic choice but shouldn't be done too often

**Review by jknpj — DISMISSED** [review]
Either move the cameras to another cameranet or remove them or the cheeseable loot from their view.

Also adding *Unable to Chuck* until Sneed is gone.

**Comment by Kromkar** [comment]
Okay, I finally figured out the issue with the map file. This will both fix not being able to open it in DM or SDMM as well as correct all issues brought to my attention, and a remap to make it less cheesable by skipping the mean boss monsters
Prism
.

**Pull request merged**